### PR TITLE
Documentation for running BigchainDB in docker containers on mac

### PR DESCRIPTION
**Pull request by krish7919**

### CODE REVIEW

File: docs/server/source/appendices/docker-on-mac.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Run BigchainDB with Docker On Mac
+
+**NOT for Production Use**
+
+Those hacking on Mac can follow this document to run BigchainDB.
```
Comment (ttmc) [inline docs/server/source/appendices/docker-on-mac.md]: Please use a word other than "hacking."

File: docs/server/source/appendices/docker-on-mac.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Running BigchainDB on Mac(Docker or otherwise) is not officially supported
+currently.
```
Comment (ttmc) [inline docs/server/source/appendices/docker-on-mac.md]: Unless Apple returns to the server market, we will not be supporting running BigchainDB Server on Macs.

File: docs/server/source/appendices/run-with-docker.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ docker run \
   --interactive \
   --rm \
   --tty \
-  --volume "$HOME/bigchaindb_docker:/data" \
+  --volume $HOME/bigchaindb_docker:/data \
```
Comment (ttmc) [inline docs/server/source/appendices/run-with-docker.md]: I wonder why you removed all the double quotes from lines like this. Was it not working?

File: docs/server/source/appendices/docker-on-mac.md (hunk 1 above)
Comment (krish7919) [inline docs/server/source/appendices/docker-on-mac.md]: Will update wording.

File: docs/server/source/appendices/run-with-docker.md (hunk 3 above)
Comment (krish7919) [inline docs/server/source/appendices/run-with-docker.md]: There were double quotes in some places and missing in others, so I just removed it everywhere to make it uniform.

**Comment by ttmc** [comment]
I'll approve this PR but please wait for @sbellem to review it as well, since he's more familiar with Docker on Macs.

**Comment by ttmc** [comment]
@krish7919 Can we merge this pull request?

**Comment by krish7919** [comment]
@ttmc: I think I dropped the ball on this one. Will merge as soon as the tests are run.
Thanks for the updating the branch! :)

**Pull request merged**